examples/apps/quick_tap.py

```diff
@@ -67,13 +67,13 @@ def __init__(self, robot: cozmo.robot.Robot):
 
         self.quick_tap_state = CHOOSE_CUBES_STATE
 
-    async def cozmo_ready_position(self):
-        self.robot.set_lift_height(0, in_parallel = True)
-        self.robot.set_head_angle(degrees(0), in_parallel = True)
+    async def move_cozmo_to_ready_pose(self):
+        await self.robot.set_lift_height(0, in_parallel = True)
+        await self.robot.set_head_angle(degrees(0), in_parallel = True).wait_for_completed()
```

Comment:
you can't just wait on the 2nd action - the 1st action could take longer, so you need to do either:
```python
action1 = self.robot.set_lift_height(0, in_parallel = True)
action2 = self.robot.set_head_angle(degrees(0), in_parallel = True)
await action1.wait_for_completed()
await action2.wait_for_completed()
```
or
```python
self.robot.set_lift_height(0, in_parallel = True)
self.robot.set_head_angle(degrees(0), in_parallel = True)
await self.robot.wait_for_all_actions_completed()
```